assets/queries/openAPI/general/global_security_field_undefined/query.rego

```diff
@@ -16,7 +16,8 @@ CxPolicy[result] {
 		"documentId": doc.id,
 		"searchKey": searchKey[version],
 		"issueType": "MissingAttribute",
-		"keyExpectedValue": "A default security schema should be defined",
-		"keyActualValue": "A default security schema is not defined",
+		"keyExpectedValue": "A default security scheme should be defined",
+		"keyActualValue": "A default security scheme is not defined",
```

Comment:
I think we could use "security property" instead "security scheme", since security scheme is an object from 3.0